stix2/sources/filesystem.py

```diff
@@ -35,6 +35,85 @@ def __init__(self, stix_dir, bundlify=False):
         self.source = FileSystemSource(stix_dir=stix_dir)
         self.sink = FileSystemSink(stix_dir=stix_dir, bundlify=bundlify)
 
+    def get(self, stix_id, allow_custom=False, version=None, _composite_filters=None):
```

Comment:
None of this is specific to the FileSystem, I don't think. I'm hoping we can suitably abstract it with `*args` and `**kwargs` in the base class.